src/main/java/lotto/AutoLottoController.java

```diff
@@ -0,0 +1,73 @@
+package lotto;
+
+import lotto.domain.*;
+import lotto.util.PrizeCalculatorUtil;
+import lotto.util.StringUtil;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+import java.util.*;
+
+public class AutoLottoController {
+
+    private final InputView inputView;
+    private final ResultView resultView;
+
+    public AutoLottoController() {
+        inputView = new InputView();
+        resultView = new ResultView();
+    }
+
+    public void run() {
+        // 금액 입력
+        int lottoCount = inputView.inputPrize();
+        // 로또 구매 갯수 표시
+        resultView.displayLottoCount(lottoCount);
+
+        User user = new User();
+        // 로또 구매
+        user.buyLotto(lottoCount);
+        // 구매 로또 표시
+        displayLotto(user);
+
+        // 당첨 번호 입력
+        Lotto winningLotto = new Lotto(StringUtil.stringToList(inputView.inputWinningLotto()));
+
+        AutoLottoMatchCount autoLottoMatchCount = new AutoLottoMatchCount();
+
+        int bonusNumber = inputView.inputSpecialNumber();
+        // 추첨
+        lottery(user, winningLotto, autoLottoMatchCount, bonusNumber);
+
+        // 당첨 통계
+        winningStatistics(autoLottoMatchCount, lottoCount);
+    }
+
+    private void displayLotto(User user) {
+        for (Lotto lotto : user.getLottoList()) {
+            resultView.displayLotto(lotto);
+        }
+    }
+
+    private void lottery(User user, Lotto winningLotto, AutoLottoMatchCount autoLottoMatchCount, int bonusNumber) {
```

Comment:
`AutoLottoMatchCount autoLottoMatchCount = new AutoLottoMatchCount();` 행위를 lottery()에서 수행하고 반환해주는건 어떨까요?
```suggestion
    private AutoLottoMatchCount lottery(User user, Lotto winningLotto, AutoLottoMatchCount autoLottoMatchCount, int bonusNumber) {
```